### PR TITLE
Builds on 5.7rc1 fail; this fixes it

### PR DESCRIPTION
**Pull request by erratic-c-programmer**

### CODE REVIEW

File: fthd_v4l2.c [hunk 1]
```diff
@@ -41,6 +41,12 @@
 #define FTHD_MIN_HEIGHT 240
 #define FTHD_NUM_FORMATS 2 /* NV16 is disabled for now */
 
+/* This trick taken from
+ * https://github.com/umlaeute/v4l2loopback/commit/4815caf32e5dceb0a0cad02d9522d0a44d1defd6 */
```
Comment (patjak) [inline fthd_v4l2.c:45]: This information is better to keep in the git commit itself. No need to clutter the code with it.

**Comment by patjak** [comment]
Hi, thanks for the patches! The 5.6 fix is already taken care of (note that we don't want iomem as cached). I have a minor comment in the 5.7 fix. Otherwise it looks fine. Would you mind reworking and sending a new pull request?

Thanks
Patrik

File: fthd_v4l2.c (hunk 1 above)
Comment (erratic-c-programmer) [reply fthd_v4l2.c]: Ok, I updated the PR to remove those comments and put them in the commit message

**Comment by patjak** [comment]
I manually picked c182db5408218dbe9f83efc04670fd82711c7389 to not get the other two and slightly changed the commit message. Thanks for the patch!

**Pull request closed without merging**